addons/hr_timesheet/hr_timesheet.py

```diff
@@ -19,79 +19,53 @@
 #
 ##############################################################################
 
-import time
+from openerp import models, api, fields, exceptions
 
-from openerp.osv import fields
-from openerp.osv import osv
 from openerp.tools.translate import _
-import openerp
-from openerp.exceptions import UserError
 
-class hr_employee(osv.osv):
-    _name = "hr.employee"
+class hr_employee(models.Model):
```

Comment:
Tip: if new api, you can use HrEmployee as class name; hr_employee is for old API. Changing API, changing coding style !